### ISSUE TITLE
device record not updated

### ISSUE THREAD

**Issue by elbidone** [meumobi/sitebuilder#349]
Today I've visited the site_id 496 through condmobi. But my device wasn't registered.
See below the log of request udpating the devices infos.

```bash
[2016-04-05 01:11:59] sitebuilder.INFO: http request {"method":"PUT","url":"/api/santamonicacondominiumclub.meumob
i.com/devices/8D80F7B1-D9EA-4300-8333-CD249161C7C0","data":{"app_version":"1.3.1","app_build":1,"model":"iPod5,1","pla
tform":"iOS","platform_version":"9.3.1","manufacturer":"Apple","push_id":"dab3d87cd36fc70b7eb655ef36da3ed16798e0fdde84
616ce8df70760f4d41ca","uuid":"8D80F7B1-D9EA-4300-8333-CD249161C7C0"},"component":"api"}
```

My login is "victor.dias@barra.com" It's saved correctly as logs below show. But if I find out the corresponding devices using user_id nothing is retrieved. Using the UUID 3 devices are retrieved but none with correct infos. 
```bash
> db.visitors.find({email: "victor.dias@barra.com"})
{ "_id" : ObjectId("5702f4879a645d1c15e8d385"), "email" : "victor.dias@barra.com", "first_name" : "Victor", "last_name" : "Dias", "site_id" : NumberLong(496), "hashed_password" : "f34bcc52074da3a81d55314077cef51574995cb9", "auth_token" : "3c03e0c2cd602551dca0e90ca122b3044901d2f3", "should_renew_password" : false, "groups" : [ "meumobi" ], "last_login" : ISODate("2016-04-04T23:11:56Z"), "created" : ISODate("2016-04-04T23:11:03Z"), "modified" : ISODate("2016-04-04T23:11:56Z") }
> db.devices.find({user_id: "5702f4879a645d1c15e8d385"})
>
> db.devices.find({uuid: "8D80F7B1-D9EA-4300-8333-CD249161C7C0"})
{ "_id" : ObjectId("56fd0ae59a645d2440adfe9c"), "uuid" : "8D80F7B1-D9EA-4300-8333-CD249161C7C0", "user_id" : "56fcf84f9a645d3b7fadfe9c", "site_id" : NumberLong(495), "push_id" : "dab3d87cd36fc70b7eb655ef36da3ed16798e0fdde84616ce8df70760f4d41ca", "model" : "iPod5,1", "manufacturer" : "Apple", "platform" : "iOS", "platform_version" : "9.2.1", "app_version" : "1.3.1", "app_build" : NumberLong(1), "created" : ISODate("2016-03-31T11:32:53Z"), "modified" : ISODate("2016-03-31T11:32:53Z") }
{ "_id" : ObjectId("56fd239b9a645df760adfe9d"), "uuid" : "8D80F7B1-D9EA-4300-8333-CD249161C7C0", "user_id" : "56fd23809a645d9463adfe9c", "site_id" : NumberLong(496), "push_id" : "dab3d87cd36fc70b7eb655ef36da3ed16798e0fdde84616ce8df70760f4d41ca", "model" : "iPod5,1", "manufacturer" : "Apple", "platform" : "iOS", "platform_version" : "9.2.1", "app_version" : "1.3.1", "app_build" : NumberLong(1), "created" : ISODate("2016-03-31T13:18:19Z"), "modified" : ISODate("2016-03-31T13:18:19Z") }
{ "_id" : ObjectId("56fd609c9a645dd412adfe9c"), "uuid" : "8D80F7B1-D9EA-4300-8333-CD249161C7C0", "user_id" : "5672e3349a645d456df2ac3b", "site_id" : NumberLong(484), "push_id" : "dab3d87cd36fc70b7eb655ef36da3ed16798e0fdde84616ce8df70760f4d41ca", "model" : "iPod5,1", "manufacturer" : "Apple", "platform" : "iOS", "platform_version" : "9.3.1", "app_version" : "1.3.1", "app_build" : NumberLong(1), "created" : ISODate("2016-03-31T17:38:36Z"), "modified" : ISODate("2016-04-04T23:03:35Z") }
>
```

**Comment by juliogreff** [comment]
This should've raise an exception, but I cannot check now because those logs are gone. Can you test again to see if this is still a problem, and provide me a bit more context from the logs?

**Comment by elbidone** [comment]
ok, I'll check it. It's pretty difficult to observe it.

**Comment by elbidone** [comment]
could be related to #374 . If true, ensure to log error when device couldn't be saved because is owned by another user.

**Comment by juliogreff** [comment]
Closed with #374.
